plugins/inputs/lotus/lotus.go

```diff
@@ -210,8 +239,8 @@ func processTipset(ctx context.Context, node api.FullNode, acc telegraf.Accumula
 	log.Println("I! Processed tipset height:", height)
 }
 
-func processHeader(ctx context.Context, acc telegraf.Accumulator, newHeader *types.BlockHeader, receivedAt time.Time) {
-	err := recordBlockHeaderPoints(ctx, acc, receivedAt, newHeader)
+func processHeader(ctx context.Context, acc telegraf.Accumulator, newHeader *types.BlockHeader, receivedAt time.Time, receivedFrom peer.ID) {
```

Comment:
`ReceivedFrom` in this context (telegraf's perception of the node) is ambiguous. (Either "the node which you received the block from" or as you intended here "the node behind telegraf which is reporting the block which was recieved".) Maybe we can avoid the ambiguity altogether with something like `seenBy` which allows the next dev to better intuit which value is the semanticly-desired value for the context.

Reply:
How does `RecordedFrom` sound? As in the node we recorded the information from.

Reply:
I prefer `By` so the from/to semantics don't get in the way. Maybe `RecordedByPeerID` so we label the data with `PeerID` as well, instead of having yet another CID in our schema.